.gitignore

```diff
@@ -1,23 +1,27 @@
-*.py[cod]
-node_modules
+# Girder repo
+!.gitignore
+
+# Build outputs
+build/
 docs/_build
 docs/_output
-.coverage
-.coverage.*
-*.egg-info
-.DS_Store
-.*.swp
-.*.swo
-.vagrant
 girder-*.tar.gz
+node_modules/
+*.egg-info
+
+# Toolchain generated
 __pycache__
-htmlcov
-### PyCharm ###
-.idea/
-.PyCharm50/
+.cache
 .eslintcache
 /npm-debug.log
-**/*.retry
-.cache
 .tox
-build/test/tox
+.vagrant
+*.py[cod]
+**/*.retry
+
+# IDE / OS generated
+.DS_Store
+.idea/
+.PyCharm50/
+.*.swo
+.*.swp
```

Comment:
Not really related to this PR, but we should remove all the platform specific stuff from this file.